app/helpers/sequence_helper.rb

```diff
@@ -0,0 +1,6 @@
+module SequenceHelper
+  def activity_player_sequence_url(sequence)
+    sequence_api_url = "#{api_v1_sequence_url(@sequence.id)}.json"
+    return  "#{ENV['ACTIVITY_PLAYER_URL']}/?sequence=#{CGI.escape(sequence_api_url)}&preview"
+  end
```

Comment:
Would it be best to consolidate this with the very similar `activity_player_url` method in the lightweight activity helper? If so, where would be the best place for the consolidated method? The application helper maybe?

Reply:
I think it is OK to leave these separated.